### ISSUE TITLE
💡 A11y: always preset font and background color

### ISSUE THREAD

**Issue by deleonio** [public-ui/kolibri#5711]
No element should be used without a background and font color whose contrast ratio has not been checked. By initially setting the background color to white and the font color to black, the contrast ratio is ensured and explicit adjustment is forced.

**Comment by deleonio** [comment]
Ja, diese Änderung erzwingt die Behandlung der Hintergrundfarbe im Theming.

https://github.com/public-ui/kolibri/pull/5703#discussion_r1417015392
